pkg/rtc/participant.go

```diff
@@ -919,7 +919,11 @@ func (p *ParticipantImpl) SubscriptionPermissionUpdate(publisherID livekit.Parti
 	}
 }
 
-func (p *ParticipantImpl) UpdateRTT(rtt uint32) {
+func (p *ParticipantImpl) UpdateRTT(rtt uint32, isUDP bool) {
+	if !isUDP {
+		p.TransportManager.UpdateRTT(rtt, false)
+		return
+	}
```

Comment:
following up on my comment about separate APIs, these functions would be straightforward and cleaner. Not that this is complex 😄 , but advocating for specific functions.